frontend/src/components/forms/SvgButton.tsx

```diff
@@ -36,15 +37,17 @@ function svgButton<E = {}>(
     ...defaults
   }: ExtraProps & { [K in keyof E]?: E[K] },
 ): SvgButton<Pick<E, Exclude<keyof E, keyof ExtraProps>>> {
-  return ({ href, ...props }) => {
+  return ({ href, disabled, onClick, ...props }) => {
     const icon = (
       <Icon
         className={classes(css.svgButtonIcon, className)}
         style={{ '--icon--color': iconColor, '--icon-hover-color': hoverColor }}
+        onClick={disabled ? undefined : onClick}
         {...defaults}
         {...props}
       />
     );
+    if (disabled) return <div className={classes(css.disabled)}>{icon}</div>;
     return href ? <a href={href}>{icon}</a> : icon;
```

Comment:
```suggestion
    return href && !disabled ? <a style={{ lineHeight: 1 }} href={href}>{icon}</a> : icon;
```

The wrapping div should not be needed, or am I missing something?

Also, I'd say that `line-height: 1` for the anchor is better than the `height: width + 2 * 1px; top: 1px;` dance.
Then the width / height should stay the same if anchor or not.

Reply:
You are right - I was probably confused by MilestonesEditor's forgotten svg styles.